common/services/move.js

```diff
@@ -136,7 +141,7 @@ const moveService = {
           'filter[to_location_id]': toLocationId,
           'filter[supplier_id]': supplierId,
         },
-        isUndefined
+        isEmpty
```

Comment:
I have a feeling this might have been using `isUndefined` for a specific reason...might have been something about retaining the ability to send an empty string, or `null` for particular relationships...

Reply:
But if that were the case, I'd have expected there to be a test, so maybe it's fine.

Reply:
plus the e2e tests still work fine...

I would suggest that the tests were simply missing... since it allowed an empty array to be passed which ended up as an empty string which ends up as a filter by value '' => no results

I think we need to move all the argument formation and sanitising to move#getAll